charts/prometheus-adapter/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: prometheus-adapter
-version: 4.11.0
+version: 4.11.1
```

Comment:
New feature should bump minor version. So 4.12.0

Reply:
Thanks @steven-sheehy 
Fixed